src/main/java/teammates/common/datatransfer/questions/FeedbackMsqQuestionDetails.java

```diff
@@ -30,7 +30,7 @@
 import teammates.ui.template.InstructorFeedbackResultsResponseRow;
 
 public class FeedbackMsqQuestionDetails extends FeedbackQuestionDetails {
-    private int numOfMsqChoices;
+    private transient int numOfMsqChoices; //NOPMD
```

Comment:
Suppress the specific rule and add a comment? E.g.
https://github.com/TEAMMATES/teammates/blob/e32eeff945bc2619d44b3a355e07ac4ad60e0edc/src/main/java/teammates/storage/entity/FeedbackSession.java#L28-L32

Reply:
done!